docs/getting-started-on-snowplow-bdp-enterprise/setup-guide-azure/index.md

```diff
@@ -1,8 +1,86 @@
 ---
 title: "Setup Snowplow BDP on Azure"
+date: "2024-01-11"
 sidebar_position: 30
 ---
 
-## Coming soon
+## Request your pipeline through Snowplow BDP Console
 
-If you are interested in deploying [BDP Enterprise](https://snowplow.io/pricing/) in your Azure account, please [join our waiting list](https://snowplow.io/bdp-enterprise-on-azure/).
+Login to Snowplow BDP Console where you'll be able to follow a step-by-step guide to getting started (including the steps below).
+
+## Setting up your Azure account
+
+To get your cloud environment ready for your Snowplow pipeline to be installed:
+
+### Create a new account
+
+Please create a new Azure account and tenant segregated from anything else you have running in Azure. For the latest documentation on setting up a new account and tenant please refer to the Azure documentation [here](https://azure.microsoft.com/).
+
+
+### Enable billing for the account
+
+Please enable billing in the tenant by creating a subscription as failing to do so will prevent the pipeline from being installed. For details on enabling billing for your tenant, please refer to the Azure documentation for [subscriptions](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription).
```

Comment:
```suggestion
Please enable billing in the tenant by creating a subscription. Otherwise, the pipeline will fail to deploy. For details on enabling billing for your tenant, please refer to the Azure documentation for [subscriptions](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription).
```